tests/sample_data/link_extractor/sgml_linkextractor_no_href.html

```diff
@@ -0,0 +1,25 @@
+<html>
+  <head>
+    <base href='http://example.com' />
+    <title>Sample page with anchor tags containing no href attribute, to test the TextResponse.follow_all method</title>
```

Comment:
why is this file called sgml_linkextractor_no_href? sgml link extractor is deprecated.

Reply:
True, I didn't realize that. I copied the test from [`scrapy.tests.test_http_response.BaseResponseTest._links_response`](https://github.com/scrapy/scrapy/blob/1.8.0/tests/test_http_response.py#L176-L179), but that doesn't seem to be specific to the `SgmlLinkExtractor`. I'll update the names, thanks for noticing.